tests/lean/run/bv_decide_rewriter.lean

```diff
@@ -292,6 +304,10 @@ example {x y z : BitVec 64} : ~~~(x &&& (y * z)) = (~~~x ||| ~~~(z * y)) := by
 
 example {x : BitVec 16} : (x = BitVec.allOnes 16) → (BitVec.uaddOverflow x x) := by bv_decide
 
+example {x : BitVec 16} : (x = BitVec.allOnes 16) → (BitVec.umulOverflow x x) := by bv_decide
+
+example {x : BitVec 8} : (x = -32#8) → (BitVec.smulOverflow x x) := by bv_decide
+
```

Comment:
I feel 1-2 examples where we expect no overflow would be useful, e.g.

`example {x : BitVec 8} : (x = 0#8) → (\not BitVec.smulOverflow x x) := by bv_decide`

Reply:
done!